precompiles/outposts/osmosis/tx.go

```diff
@@ -126,6 +128,7 @@ func (p Precompile) Swap(
 	}
 	packetString := packet.String()
 
+	timeoutTimestamp := ctx.BlockTime().Add(ics20.DefaultTimeoutMinutes * time.Minute).Unix()
```

Comment:
Should be nanoseconds
```suggestion
	timeoutTimestamp := ctx.BlockTime().Add(ics20.DefaultTimeoutMinutes * time.Minute).UnixNano()
```